src/cloud-api-adaptor/podvm-mkosi/Dockerfile.podvm

```diff
@@ -0,0 +1,44 @@
+# Adapted from https://github.com/kubernetes-sigs/kind/blob/main/images/base/Dockerfile
```

Comment:
It's better move this Dockerfile to podvm/Dockerfile_podvm_docker to align with other distro.
I guess we'll move all Dockerfile_xx_xx from podvm to podvm-mkosi after sunset pakcer, we can do it for all distro at same time to avoid confusion.

Reply:
It does make sense to have all dockerfiles in podvm, but I was in two minds.
Let's do it once we switch to single distro + method to avoid confusion.
@huoqifeng wdyt ?

Reply:
Sure @bpradipt , we can make the change later.